Libraries/LibGfx/PainterSkia.cpp

```diff
@@ -60,6 +64,198 @@ static SkPathFillType to_skia_path_fill_type(Gfx::WindingRule winding_rule)
     VERIFY_NOT_REACHED();
 }
 
+static SkPoint to_skia_point(auto const& point)
```

Comment:
We have this in DisplayListPlayerSkia.cpp
To avoid code duplication, can this be moved to LibGfx/SkiaUtils.h or another common header?